locations/spiders/rocomamas.py

```diff
@@ -0,0 +1,118 @@
+from scrapy.http import JsonRequest
+
+from locations.categories import Extras, apply_yes_no
+from locations.hours import DAYS_FULL, OpeningHours
+from locations.items import SocialMedia, set_social_media
+from locations.json_blob_spider import JSONBlobSpider
+
+ROCOMAMAS_COUNTRIES = {
+    "BW",
+    "CD",
+    "GH",
+    "KE",
+    "IN",
+    "NA",
+    "SA",
+    "SZ",
+    "ZM",
+    "ZA",
+    "ZW",
+}
+
+ROCOMAMAS_BRANDS = {
+    "ROCO": {"brand": "RocoMamas", "brand_wikidata": "Q123034964"},
+    "ROCOGO": {"brand": "RocoGo", "brand_wikidata": "Q123034964"},
+}
+
+SOCIAL_MEDIA_MAP = {
+    "Facebook": SocialMedia.FACEBOOK,
+    "TripAdvisor": SocialMedia.TRIPADVISOR,
+}
+
+
+class RocomamasSpider(JSONBlobSpider):
+    name = "rocomamas"
+    start_urls = ["https://rocomamas.com/api/proxy"]
+
+    def start_requests(self):
+        for url in self.start_urls:
+            for country in ROCOMAMAS_COUNTRIES:
+                body_data = (
+                    '{"url":"restaurants?brandKey=ROCO,ROCOGO&countryCode='
+                    + country
+                    + '&tradingStatus=Open&filterHidden=true&expand=channels","method":"GET"}'
+                )
+                yield JsonRequest(
+                    url=url,
+                    headers={"Content-Type": "application/json"},
+                    method="POST",
+                    body=body_data,
```

Comment:
Consider using data instead of body

Reply:
Thanks, although it seems that it doesn't work if I use data here